### PR TITLE
Add notebooks for plotting JWST SI WFE, and JWST SI MIMF field points

### PR DESCRIPTION
**Pull request by mperrin**
Adds the notebook which makes the [plots of SI WFE included in the docs](https://webbpsf.readthedocs.io/en/latest/jwst.html#si-wfe). 

Also adds the notebook which made the MIMF field points chart that we used in commissioning (independent from but somewhat related to that first notebook). Not sure this will ever be needed again but I realized it was missing from the git repo, along with the SI WFE plots notebook.

These are both added in the `dev_utils` subdirectory, since these are more specialized / less general purpose, and not too likely to be used by typical webbpsf users.

### CODE REVIEW

**Comment by mperrin** [comment]
Reminder this is still waiting on a review from ~months ago, fyi @obi-wan76

File: dev_utils/Plot SI WFE Field Dependence.ipynb
Comment (marcio-melendez) [inline dev_utils/Plot SI WFE Field Dependence.ipynb:1]: This notebook runs fine except for the last cell:
`main_opd_plot(nircamswa, verbose=True)` which should be `main_opd_plot_isimcv(nircamswa, verbose=True)`
However, when running it returns an error:

> ValueError: Detector pixel coordinates must be nonnegative integers

I think because `nircamswa._tel_coords() = [1.1194241, −6.7511693]′`

File: dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb
Comment (marcio-melendez) [inline dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb:1]: `webbpsf.Instrument` should be `webbpsf.instrument`
The file MIMF_FP_Table_From_WAG.xlsx is missing

**Review by marcio-melendez — CHANGES_REQUESTED** [review]
A couple of small comments for each notebook

**Comment by mperrin** [comment]
I added the missing excel reference data file. 

For the plotting of the OPDs for NIRCam coronagraphy, that has never actually worked. The coronagraphy WFE for NIRCam is handled differently because of the Lyot wedges. I may try to make this plotting also work (it's arguably relevant for some science) but it won't happen right away

File: dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb
Comment (marcio-melendez) [reply dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb:1]: I see the file, there is still the change from `webbpsf.Instrument `to `webbpsf.instrument`.

**Comment by mperrin** [comment]
I fixed (enhanced) the plotting to also work for NIRCam coronagraphy. This ended up being a semi-substantial bit of work, requiring modifications throughout several parts of the notebook. Working now for me.

File: dev_utils/Plot SI WFE Field Dependence.ipynb
Comment (marcio-melendez) [reply dev_utils/Plot SI WFE Field Dependence.ipynb:1]: This notebook is running in its entirely for me, attach the last figure in NRC Coron LW
image

Comment (mperrin) [reply dev_utils/Plot SI WFE Field Dependence.ipynb:1]: Great. FYI, I showed this output to @JarronL and he confirmed it looked good (correct) to him.

Comment (mperrin) [reply dev_utils/Plot SI WFE Field Dependence.ipynb:1]: btw I think you still have to mark it approved in a new review before either of us can merge it

File: dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb
Comment (marcio-melendez) [reply dev_utils/Plot SI MIMF Field Point Locations Wall Chart.ipynb:1]: @mperrin there is still the pending comment on this notebook, the wall chart, about changing  the instrument class from upper case to lower case.

**Review by marcio-melendez — APPROVED** [review]
Looks good, comments in both notebooks have been resolved. Great addition.

**Pull request merged**
